scripts/install_third_party.sh

```diff
@@ -60,6 +60,11 @@ if [ ! -d "$NODE_PATH" ]; then
   rm node-download.tgz
 fi
 
+# List all node modules that are currently installed. The "npm list" command is
```

Comment:
Nah -- I think lines 24 to 61 should be transferred to setup.sh instead. The reason is that NPM_INSTALLED_MODULES is used within a function at the bottom of setup.sh, so it seems weird to declare it only here -- it ought to be in setup.sh or a 'predecessor' of it. There are other scripts that use that function (install_node_module()) -- and they may or may not call install_third_party.sh.